mp4/eventmessage.go

```diff
@@ -166,15 +166,17 @@ func DecodeSilbSR(hdr BoxHeader, startPos uint64, sr bits.SliceReader) (Box, err
 	b.Version = uint8(versionAndFlags >> 24)
 	b.Flags = versionAndFlags & flagsMask
 	nrSchemes := sr.ReadUint32()
-	b.Schemes = make([]SilbEntry, nrSchemes)
```

Comment:
I tried to implement a size check here but the implementation of Size() does not take into account the size of `b.Schemes`:

```
func (b *SilbBox) Size() uint64 {
	size := uint64(boxHeaderSize + 8)
	for _, scheme := range b.Schemes {
		size += uint64(len(scheme.SchemeIdURI) + 1 + len(scheme.Value) + 1 + 1)
	}
	size += 1 // OtherSchemesFlag
	return size
}
```